src/_data/sites/ja.yml

```
@@ -0,0 +1,544 @@
+---
+#------------------------------------------------------------------------------
+# English Site Details
+# The primary ESLint site that is hosted at eslint.org
+# Author: Nicholas C. Zakas
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: ja
+  flag: 🇯🇵
+  name: Japanese
+locale: ja-JP
+hostname: ja.eslint.org
+
+#------------------------------------------------------------------------------
+# Analytics
+#------------------------------------------------------------------------------
+
+google_analytics:
+  code: "UA-60915033-1"
+
+#------------------------------------------------------------------------------
+# Ads
+#------------------------------------------------------------------------------
+
+carbon_ads:
+  serve: CESDV2QM
+  placement: eslintorg
+
+#------------------------------------------------------------------------------
+# Shared
+#------------------------------------------------------------------------------
+
+shared:
+  get_started: 始める
+  become_a_sponsor: スポンサーになる
+  eslint_logo_alt: ESLintのロゴ
+  description: >
+    JavaScriptのパターンを識別し報告するための、プラグイン可能で設定可能なリンターツールです。
+    コードの品質を簡単に維持することができます。
+  title_format: PAGE_TITLE - ESLint - プラグイン可能なJavaScriptリンターツール
+  skip_to_content: 本文へスキップ
+
+#------------------------------------------------------------------------------
+# Navigation
+#------------------------------------------------------------------------------
+
+navigation:
+  - text: チーム
+    link: team
+  - text: ブログ
+    link: blog
+  - text: ドキュメンテーション
+    link: docs
+  - text: ストアー
+    link: store
+    target: _blank
+  - text: Playground
+    link: playground
+    target: _blank
+
+#------------------------------------------------------------------------------
+# Footer
+#------------------------------------------------------------------------------
+
+footer:
+  title: JavaScriptのコードを修正する準備はできましたか？
+  description: npmからインストール。または今日から寄付を開始する。
+  secondary: Secondary
+  social_icons:
+    title: ソーシャルメディア
+    twitter: ツイッター
+    chat: ディスコード
+    mailing_list: Google グループ
+    github: ギットハブ
```

The service name is often written in its original notation.

https://ja.wikipedia.org/wiki/Discord
https://ja.wikipedia.org/wiki/Twitter
https://ja.wikipedia.org/wiki/GitHub

```suggestion
    twitter: Twitter
    chat: Discord
    mailing_list: Google グループ
    github: GitHub
```